.gitignore

```diff
@@ -101,4 +101,6 @@ env.bat
 
 # Subprojects
 subprojects/*
+!subprojects/packagefiles
+!subprojects/packagefiles/*
```

Comment:
Why ignoring package files if this PR addsit?

Reply:
My bad, that shouldn't be committed.

Reply:
ohhhhh, right right, I didn't note the `!`, thanks.